vispy/testing/_runners.py

```diff
@@ -111,6 +111,25 @@ def _unit(mode, extra_arg_string, coverage=False):
         os.rename(op.join(cwd, '.coverage'), out_name)
 
 
+def _docs():
+    """test docstring paramters
+    using vispy/utils/tests/test_docstring_parameters.py"""
+    print ("warning: this imports vispy from the vispy installation. it is"
```

Comment:
I'm not super happy with the solution. Is there a way to ensure that the newest versions are automatically tested? This was the simplest solution I could come up with (i.e, import the test from the vispy install), but I think there's a better solution that I'm missing